src/colordistancesensor.ts

```diff
@@ -0,0 +1,25 @@
+import { Device } from "./device";
+import { Hub } from "./hub";
+
+import * as Consts from "./consts";
+
+export class ColorDistanceSensor extends Device {
+
+    constructor (hub: Hub, portId: number) {
+        super(hub, portId, Consts.DeviceType.COLOR_DISTANCE_SENSOR);
+
+        this.on("newListener", (event) => {
+            switch (event) {
+                case "color":
+                    this.subscribe(0x00);
```

Comment:
I wonder if we should avoid using hex notation as all const are in decimal and I see modes as almost const.

Reply:
In this case I don't mind either way. But for `Consts.DeviceType` I would actually prefer to use hex literals as this is how they are listed in [LWP doc](https://lego.github.io/lego-ble-wireless-protocol-docs/index.html#io-type-id).

Reply:
It's a good question. I think for the moment having the values hardcoded is a placeholder, I'd like to put all consts in a file somewhere. So - hex or decimal? Perhaps hex as @nutki suggests?

Reply:
Yes, like @nutki wrote hex make more sense regarding the doc. Even more if we had some const with all message headers and all.

Plus, the end users can use whatever format they want when they call methods.

Reply:
@aileo I've started getting rid of the hex notation and implementing modes as enums. And as suggested by @nutki somewhere else, subscriptions are automatically made when attaching the associated event handler. It's much cleaner, thanks!